apps/cli/templates/defaults/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "private": true,
+  "sideEffects": false,
+  "scripts": {
+    "build": "remix build",
+    "dev": "remix dev",
+    "typecheck": "tsc"
+  },
+  "dependencies": {
+    "@remix-run/css-bundle": "^1.17.1",
+    "@remix-run/node": "^1.17.1",
+    "@remix-run/react": "^1.17.1",
+    "@webstudio-is/react-sdk": "^0.82.0",
+    "@webstudio-is/sdk-components-react-remix": "^0.82.0",
+    "@webstudio-is/sdk-components-react": "^0.82.0",
+    "@webstudio-is/form-handlers": "^0.75.0",
```

Comment:
This is tricky too. Need to maintain these versions to match latest published ones.

Reply:
In one way, in will help in making cli independent of the updates to the packages. And keep always be in sync. But in another context, this can actually help in tying up the `cli` version and the `sdk` version together. If there are any breaking changes in `sdk` the cli will pul the latest ones. And get's effected with the pre-exiting projects.
So, a new version of `cli` can be maintained every time the `@webstudio/` packages are released